src/mugs.js

```diff
@@ -856,8 +863,25 @@ define([
             // could use a key-value widget for this in the future
             rawBindAttributes: {
                 presence: 'optional',
-                lstring: 'Extra Bind Attributes',
-            }
+                lstring: 'Extra Bind Attributes'
+            },
+            setValue: {
```

Comment:
I was wavering on doing this since it's called setvalue in the xml, but it does get messy with those references and widget.setValue, so I changed it.